projects/app/src/data/mistakes.ts

```
@@ -0,0 +1,90 @@
+}
+
+export function hanziToPinyinQuestionMistakes(
+  question: Pick<HanziToPinyinQuestion, `answers` | `skill`>,
+  userAnswer: readonly string[],
+): MistakeType[] {
+  const mistakes: MistakeType[] = [];
+
+  const firstAnswer = question.answers[0];
+  invariant(firstAnswer != null, `expected answer to be defined`);
+  const answer: readonly PinyinSyllable[] | undefined =
+    typeof firstAnswer === `string` ? [firstAnswer] : firstAnswer;
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)